zipkin-server/src/test/java/zipkin/server/internal/ITZipkinMetricsHealth.java

```diff
@@ -103,43 +103,48 @@ static List readJson(String json) throws Exception {
     String prometheus = response.body().string();
 
     assertThat(prometheus)
-      .contains("http_request_duration_seconds_count{path=\"/api/v1/spans\",method=\"POST\",} 2.0");
+      .contains("counter_zipkin_collector_spans_http_total " + (messagesCount + 6));
```

Comment:
did this path exact match beforehand?

Reply:
I need to implement the `http_request_duration_seconds_count`

Reply:
Yeah. I have just completed that as well. Doing the clean up and updating the tests now.

Reply:
@shakuzen Would love to see that undertow filter in micrometer-core.